cobalt/cobalt.cc

```diff
@@ -45,41 +54,109 @@ content::ContentMainRunner* GetContentMainRunner() {
 static base::AtExitManager* g_exit_manager = nullptr;
 static cobalt::CobaltMainDelegate* g_content_main_delegate = nullptr;
 static PlatformEventSourceStarboard* g_platform_event_source = nullptr;
+
+// Toggle-only Switches.
+const auto kCobaltToggleSwitches = std::to_array<const char*>({
+  // Disable first run experience, kiosk, etc.
+  "disable-fre",
+  switches::kNoFirstRun,
+  switches::kKioskMode,
+  // Enable Blink to work in overlay video mode
+  switches::kForceVideoOverlays,
+  // Disable multiprocess mode.
+  switches::kSingleProcess,
+  // Accelerated GL is blanket disabled for Linux. Ignore the GPU blocklist
+  // to enable it.
+  switches::kIgnoreGpuBlocklist,
+  // This flag is added specifically for m114 and should be removed after
+  // rebasing to m120+
+#if BUILDFLAG(IS_ANDROID)
+  switches::kUserLevelMemoryPressureSignalParams,
```

Comment:
Are these flags used on Android? I thought those flags were in CobaltActivity.java

Reply:
Not sure about the java-side of things. I found that this flag is defined only for android platforms when trying to get the header includes: https://github.com/youtube/cobalt/blob/main/content/public/common/content_switches.h#L266

Perhaps @johnxwork might be able to shed some light?